ballerina/vector_store.bal

```diff
@@ -83,48 +89,65 @@ public isolated class VectorStore {
 
     # Deletes vector entries from the store by their reference document ID.
     #
-    # + id - The ID of the vector entry to delete.
-    # + return - An `ai:Error` if the deletion fails; otherwise, `()` is returned indicating success.
-    public isolated function delete(string id) returns ai:Error? {
+    # + ids - The ID/s of the vector entries to delete
+    # + return - An `ai:Error` if the deletion fails; otherwise, `()` is returned indicating success
+    public isolated function delete(string|string[] ids) returns ai:Error? {
         lock {
-            int|error index = int:fromString(id);
-            if index is error {
-                return error ai:Error("failed to convert id to int", index);
+            if ids is string {
+                return self.deleteEntry(ids);
             }
-            int|error deleteResult = self.milvusClient->delete({
-                collectionName: self.config.collectionName,
-                ids: [index]
-            });
-            if deleteResult is error {
-                return error("failed to delete vector entry", deleteResult);
+            foreach string id in ids.cloneReadOnly() {
+                return self.deleteEntry(id);
```

Comment:
lets check whether there is a batch delete API provided by milvus. if so we can use that here.

Reply:
Fixed in [ec3d53a](https://github.com/ballerina-platform/module-ballerinax-ai.milvus/pull/8/commits/ec3d53a6469ab718121bf1f7e57350be2219aa81)